libraries/autoload.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
To anyone that might ask why these composer files are included is because Backdrop has its own release workflow that retrieves and builds the module when a release is published. I looked for other modules that has external libraries being used and they all included the library within their repository. Here are some of them:
- https://github.com/backdrop-contrib/colorbox
- https://github.com/backdrop-contrib/repeating_dates
- https://github.com/backdrop-contrib/leaflet